Feat/single tech/feature engineering on fabric br/setup env notebook (#654)
Azure-Samples/modern-data-warehouse-dataops: single_tech_samples/fabric/feature_engineering_on_fabric/README.md

- Access to an Azure subscription. You can sign up for a free trial by following [this link](https://azure.microsoft.com/en-us/free/).
- Permissions to [create a service principal](https://learn.microsoft.com/entra/identity-platform/howto-create-service-principal-portal#register-an-application-with-microsoft-entra-id-and-create-a-service-principal) in Microsoft Entra ID.

> 

Reviewer: ```suggestion
```

  For granting access to Microsoft Purview access, assign [Data curators](https://learn.microsoft.com/en-us/purview/how-to-create-and-manage-collections#roles) role in the Purview root collection to this SP. For details, refer to [Microsoft documentation](https://learn.microsoft.com/purview/how-to-create-and-manage-collections#add-role-assignments).

> 

Reviewer: ```suggestion
```

> #### [Optional] Notebook for Creating Above Resources

Reviewer: ```suggestion
```

> To streamline your setup process, utilize the [deploy-azure-resources.ipynb](./src/infra/deploy-azure-resources.ipynb) notebook. This tool is designed to guide you through the creation of Azure resources mentioned above in an interactive and user-friendly way.

Reviewer: ```suggestion
> Note: An optional (and novel) alternative for deploying the specified Azure resources involves utilizing the Fabric notebook. The [deploy-azure-resources.ipynb](./src/infra/deploy-azure-resources.ipynb) notebook can be used to interactively create the aforementioned resources. It utilizes Azure's Python SDK along with Azure CLI to facilitate the resource creation process.
```

> 
> #### [Optional] Notebook for Creating Above Resources
> To streamline your setup process, utilize the [deploy-azure-resources.ipynb](./src/infra/deploy-azure-resources.ipynb) notebook. This tool is designed to guide you through the creation of Azure resources mentioned above in an interactive and user-friendly way.

Reviewer: ```suggestion
> Note: *An optional (and novel) alternative for deploying the specified Azure resources involves utilizing the Fabric notebook. The [deploy-azure-resources.ipynb](./src/infra/deploy-azure-resources.ipynb) notebook can be used to interactively create the aforementioned resources. It utilizes Azure's Python SDK along with Azure CLI to facilitate the resource creation process.*
```
